e2e/support/helpers/e2e-ad-hoc-question-helpers.js

```diff
@@ -110,18 +110,35 @@ export function openTable({
   );
 }
 
+/**
+ *
```

Comment:
We probably don't want the SQL sidebar to go out of view when scrolling query builder contents.

https://github.com/metabase/metabase/assets/6830683/b79d66dd-44d2-4a9f-bc92-67e9252a1d5a

Reply:
https://github.com/metabase/metabase/issues/40697